build.sbt

```diff
@@ -457,142 +392,23 @@ lazy val app        = project
-      catsCore,
-      catsEffect,
-      circeCore,
-      circeGenericExtras,
-      logback,
-      monixEval,
-      pureconfig,
-      scalaLogging,
-      akkaHttpTestKit % Test,
-      akkaTestKit     % Test,
-      mockito         % Test,
-      scalaTest       % Test
-    ),
-    cleanFiles              ++= Seq(
-      baseDirectory.value / "permissions-fixer" / "target" / "**",
-      baseDirectory.value / "nexus-storage.jar"
-    ),
-    Test / testOptions       += Tests.Argument(TestFrameworks.ScalaTest, "-o", "-u", "target/test-reports"),
-    Test / parallelExecution := false,
-    mappings in Universal    := {
-      (mappings in Universal).value :+ cargo.value
-    }
-  )
-
-lazy val delta = project
-  .in(file("delta"))
-  .dependsOn(sourcing, rdfOld)
-  .enablePlugins(JmhPlugin, BuildInfoPlugin, UniversalPlugin, JavaAppPackaging, DockerPlugin)
-  .settings(shared, compilation, assertJavaVersion, coverage, release, servicePackaging)
-  .settings(
-    name                 := "delta",
-    moduleName           := "delta",
-    coverageMinimum      := 75d,
-    buildInfoKeys        := Seq[BuildInfoKey](version),
-    Docker / packageName := "nexus-delta",
-    buildInfoPackage     := "ch.epfl.bluebrain.nexus.delta.config"
-  )
-  .settings(kamonSettings)
-  .settings(
-    libraryDependencies       ++= Seq(
-      akkaClusterSharding,
-      akkaHttp,
-      akkaHttpCirce,
-      akkaHttpCors,
-      akkaPersistence,
-      akkaPersistenceCassandra,
-      akkaPersistenceQuery,
-      akkaSlf4j,
-      alleycatsCore,
-      alpakkaS3,
-      alpakkaSse,
-      catsCore,
-      catsEffectRetry,
-      catsEffect,
-      guava,
-      jenaArq,
-      jsonldjava,
-      kryo,
-      logback,
-      magnolia,
-      monixEval,
-      nimbusJoseJwt,
-      parboiled2,
-      topBraidShacl,
-      akkaHttpTestKit         % Test,
-      akkaPersistenceInMem    % Test,
-      akkaPersistenceLauncher % Test,
-      akkaTestKit             % Test,
-      asm                     % Test,
-      circeLiteral            % Test,
-      distageDocker           % Test,
-      distageTestkit          % Test,
-      jsonldjava              % Test,
-      log4jCore               % Test,
-      log4jApi                % Test,
-      mockito                 % Test,
-      scalaTest               % Test
-    ),
-    sourceDirectory in Jmh     := (sourceDirectory in Test).value,
-    classDirectory in Jmh      := (classDirectory in Test).value,
-    dependencyClasspath in Jmh := (dependencyClasspath in Test).value,
-    compile in Jmh             := (compile in Jmh).dependsOn(compile in Test).value,
-    run in Jmh                 := (run in Jmh).dependsOn(Keys.compile in Jmh).evaluated,
-    Test / fork                := true
-  )
-
 lazy val root = project
   .in(file("."))
   .settings(name := "nexus", moduleName := "nexus")
   .settings(noPublish)
-  .aggregate(docs, cli, sourcing, rdfOld, testkit, storage, delta)
+  .aggregate(docs, cli, testkit)
```

Comment:
either aggregate on all sub-modules, or keep the delta project as a simple aggregate over its sub-modules